Project.toml

```diff
@@ -5,7 +5,6 @@ version = "0.1.0"
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```

Comment:
Removing this dep breaks the package:

[break.txt](https://github.com/JuliaGPU/AMDGPUnative.jl/files/3883869/break.txt)

Reply:
Sorry about that, I must've been a bit sleepy when I was working on this...